checker/lib/README

```diff
@@ -3,20 +3,25 @@ This directory contains annotated versions of libraries.
 
 Instructions for users:
 
-During type-checking, put these versions on your classpath,
+During type-checking, put these jars on your classpath,
 to obtain more precise type-checking with fewer false positive warnings.
 
+If the file for the library you need is a stub file (its extension is
+.astub instead of .jar), you can supply it to the checker by using
+the -Astubs option, like this: -Astubs=javadoc.astub.
 
 Instructions for people creating a new annotated library:
 
 If you annotate a new library, please inform the Checker Framework developers
  so that they can include it in the Checker Framework distribution.
 
-When adding a .jar file to this directory, update the below list and also
+When adding a .jar or .astub file to this directory, update the below list and also
```

Comment:
Is it also necessary to tweak the source code of the checker to know about the .astub file, or are all astub files automatically discovered?